pandas/tests/extension/list/test_list.py

```diff
@@ -16,12 +16,12 @@ def dtype():
 @pytest.fixture
 def data():
     """Length-100 ListArray for semantics test."""
-    data = make_data()
+    # TODO: make better random data
+    data = [list("a"), list("ab"), list("abc")] * 33 + [None]
+    return ListArray(data)
 
-    while len(data[0]) == len(data[1]):
-        data = make_data()
 
-    return ListArray(data)
+class TestListArray(BaseConstructorsTests): ...
```

Comment:
i think we moved away from this pattern to just ExtensionTests

Reply:
Yea - I think we can move to that as this gets more production ready. I just wanted to start with something really small while in draft mode